protocols/Illumina2GafFastQ.sh

```diff
@@ -101,37 +112,39 @@ then
 		cd "${intermediateDir}"
 		md5sum Undetermined_S[0-9]*_L00${lane}_R1_001.fastq.gz >  ${runResultsDir}/${sequencingStartDate}_${sequencer}_${run}_${flowcell}_L${lane}_DISCARDED.fq.gz.md5
 
-                cp Undetermined_S[0-9]*_L00${lane}_R1_001.fastq.gz ${runResultsDir}/${sequencingStartDate}_${sequencer}_${run}_${flowcell}_L${lane}_DISCARDED.fq.gz
+		cp Undetermined_S[0-9]*_L00${lane}_R1_001.fastq.gz ${runResultsDir}/${sequencingStartDate}_${sequencer}_${run}_${flowcell}_L${lane}_DISCARDED.fq.gz
+
+		cd  "${runResultsDir}"
 
-                cd "${runResultsDir}"
+		VAR1=${sequencingStartDate}_${sequencer}_${run}_${flowcell}_L${lane}_DISCARDED.fq.gz
```

Comment:
missing quotes ;)

Reply:
Nice, you're commenting on your own code :p